src/compose-create-pull-request.ts

```diff
@@ -57,17 +57,17 @@ export async function composeCreatePullRequest(
     const user = await octokit.request("GET /user");
 
     // https://developer.github.com/v3/repos/forks/#list-forks
-    const forks = await octokit.request("GET /repos/:owner/:repo/forks", {
+    const forks = await octokit.request("GET /repos/{owner}/{repo}/forks", {
       owner,
       repo,
     });
     const hasFork = forks.data.find(
-      (fork) => fork.owner.login === user.data.login
+      (fork) => fork!.owner!.login === user.data.login
```

Comment:
Not sure what the right approach here is since if you use `fork?.owner?.login` code coverage decreases.